src/platform/mhv/secondary-nav/sass/mhv-sec-nav.scss

```diff
@@ -1,13 +1,10 @@
-$nav-width-bar-max-width: 696px;
-$nav-mobile-breakpoint: 650px;
-$nav-abbr-breakpoint: 415px;
-
-.mhv-c-sec-nav-bar {
-  max-width: $nav-width-bar-max-width;
-}
+$nav-width-bar-max-width: 720px;
+$nav-mobile-breakpoint: 629px;
+$nav-abbr-breakpoint: 467px;
 
 .mhv-c-sec-nav-bar-row {
-  justify-content: space-evenly;
+  justify-content: space-around;
```

Comment:
Right now the spacing visually aligns the left edge of secondary nav wit the left edge of the text of the megamenu, but not the actual click/hover area

<img width="1078" alt="screenshot of mhv secondary nav with megamenu hover state active" src="https://github.com/department-of-veterans-affairs/vets-website/assets/279327/a0213a28-8982-40f1-b48f-794bd6ee336b">

whereas

```suggestion
  justify-content: space-between;
```

would get you 

<img width="1124" alt="mhv secondary nav with left edge aligned with left edge of mega menu hover" src="https://github.com/department-of-veterans-affairs/vets-website/assets/279327/6af332c1-8d5c-434d-ba77-16dc432a4fcb">


I don't actually know what's better, but thought I'd point it out. The mix of rounded (secondary nav) and hard (mega menu) edges also seems unusual to me

Reply:
Had a discussion in slack and the preference is not to align it to the edges as the text will then look off to the left.